frontend/language/src/en.json

```
@@ -631,6 +631,10 @@
   "schema_editor.info_dialog_title": "New data models page",
   "schema_editor.integer": "Integer",
   "schema_editor.keyword": "Keyword",
+  "schema_editor.language_add_language":"Add language",
+  "schema_editor.language_conferm_deletion": "Yes, delete language",
```

Small typo error here, `conferm` should be `confirm`
```suggestion
  "schema_editor.language_confirm_deletion": "Yes, delete language",
```

---

fixed